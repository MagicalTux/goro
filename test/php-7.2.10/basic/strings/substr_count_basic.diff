     ***Testing basic operations ***
002- bool(false)
003- bool(false)
004- int(0)
005- int(0)
006- int(0)
007- int(100)
008- int(200)
009- int(160)
010- int(10)
011- int(40)
012- int(10)
013- int(30)
014- Done
002+ 
003+ Fatal error: Uncaught Error: Call to undefined function substr_count in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/substr_count_basic.php:4
004+ Stack trace:
005+ #0 {main}
006+   thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/substr_count_basic.php on line 4
