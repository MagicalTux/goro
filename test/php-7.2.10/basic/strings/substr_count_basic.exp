***Testing basic operations ***
bool(false)
bool(false)
int(0)
int(0)
int(0)
int(100)
int(200)
int(160)
int(10)
int(40)
int(10)
int(30)
Done