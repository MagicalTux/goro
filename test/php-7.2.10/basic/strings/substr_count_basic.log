
---- EXPECTED OUTPUT
***Testing basic operations ***
bool(false)
bool(false)
int(0)
int(0)
int(0)
int(100)
int(200)
int(160)
int(10)
int(40)
int(10)
int(30)
Done
---- ACTUAL OUTPUT
***Testing basic operations ***

Fatal error: Uncaught Error: Call to undefined function substr_count in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/substr_count_basic.php:4
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/substr_count_basic.php on line 4
---- FAILED
