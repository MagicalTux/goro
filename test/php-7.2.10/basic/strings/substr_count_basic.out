***Testing basic operations ***

Fatal error: Uncaught Error: Call to undefined function substr_count in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/substr_count_basic.php:4
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/substr_count_basic.php on line 4